alert-common/src/main/java/com/synopsys/integration/alert/common/rest/model/CustomMessageConfigModel.java

```diff
@@ -28,38 +28,48 @@
 
 import com.synopsys.integration.alert.common.persistence.accessor.FieldAccessor;
 
-public class TestConfigModel extends AlertSerializableModel {
+public class CustomMessageConfigModel extends JobFieldModel {
```

Comment:
If global test actions are also being converted to use CustomMessageConfigModel, we should not extend JobFieldModel. Can we get this functionality while still extending AlertSerializableModel?